### DISCUSSION TITLE
Regarding Health Insurance

### DISCUSSION THREAD

**Discussion by LovleenKaur-tal** [snigi-gupta/AttendingUniversityAtBuffalo#49] [General]
Hi, 

May I know what are the best Health insurance options available for international students? 

Thanks

**Comment by snigi-gupta** [comment]
Hi @LovleenKaur-tal,

For UB students, by default, we are provided UHC health insurance and that is a pretty good one. Most students go with the default health insurance. The reason for that is that health insurance here is complicated and hence opting for a different one means a lot of paperwork, terms, and conditions, and that means you have to go through each document in detail. At times the coverage also is different. I wouldn't suggest you go down that road. I have rarely seen someone get different health insurance from what the univ is offering. 
The UHC health insurance offers most of the coverage you will need. Only eyes and dental insurance is not there. Get a full check-up done before you come to the US. Get extra spectacles/lenses if you wear them, and get a dental cleanup done.

I also wanted to highlight:
You also MUST have health insurance. It is illegal to be without health insurance technically, and if you don't have one, most hospitals don't agree to treat you. As you must be aware, health costs are super high in the US.
